docs/src/pages/index.js

```diff
@@ -1,43 +1,34 @@
 import React from 'react'
 
-import { External as Link } from '../components/link'
+import Link from '../components/link'
 import Layout from '../components/layout'
+import { H1 } from '../components/core/heading'
+import P from '../components/core/paragraph'
 
 export default () => {
   return (
     <Layout>
-      <h1>Hi people</h1>
-      <p>
-        Integrate with the Notify API using one of our clients (links open in a
-        new tab)
-      </p>
+      <H1>Documentation</H1>
+      <P>
+        Notify is a service developed by the DTA that lets government agencies
+        to send and receive text messages and email messages.
+      </P>
 
-      <ul>
-        <li>
-          <Link
-            blank
-            href="https://github.com/alphagov/notifications-java-client"
-          >
-            Java
-          </Link>
-        </li>
-        <li>
-          <Link
-            blank
-            href="https://github.com/alphagov/notifications-python-client"
-          >
-            Python
-          </Link>
-        </li>
-        <li>
-          <Link
-            blank
-            href="https://github.com/alphagov/notifications-node-client"
-          >
-            Node.js
-          </Link>
-        </li>
-      </ul>
+      <div className="prevent-next-paragraphs-from-getting-hero-text" />
```

Comment:
Random `prevent-next-paragraphs-from-getting-hero-text`?

Reply:
we've got a pretty hectic adjacent sibling selector going on here https://github.com/govau/notify/blob/master/docs/src/components/core/heading.js#L8

this is just a hack to prevent it from selecting all these following paragraphs; which I attempted to document with this stupid classname. does not seem to have served its intended purpose.